test/models/mdetr/test_text_encoder.py

```diff
@@ -10,25 +10,26 @@
 
 from test.test_utils import assert_expected, set_rng_seed
 from torchmultimodal.models.mdetr.text_encoder import (
-    MDETRTextEmbeddings,
     MDETRTextEncoder,
     ModifiedTransformerEncoder,
 )
+from torchmultimodal.modules.layers.text_embedding import BERTTextEmbeddings
 
 
 class TestMDETRTextEncoder(unittest.TestCase):
     def setUp(self):
         set_rng_seed(0)
         self.max_position_embeddings = 514
         self.hidden_size = 768
-        self.embeddings = MDETRTextEmbeddings(
+        self.embeddings = BERTTextEmbeddings(
             hidden_size=self.hidden_size,
             vocab_size=50265,
             pad_token_id=1,
             type_vocab_size=1,
             max_position_embeddings=self.max_position_embeddings,
             layer_norm_eps=1e-05,
-            hidden_dropout_prob=0.1,
+            dropout=0.1,
```

Comment:
why setting dropout in test? remove

Reply:
this was originally there, I'm inclined to leave as is so I don't have to update the test values